### PR TITLE
Updated to new tooltips

### LINKED ISSUE #6377: Redesign: Tooltip in workspace
## Feature Description

Update tooltip in workspace with that from the design system

- [x] Page menu
- [x] Carousel and menu (keyboard tooltip fixed in #6385)

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by barklund**
## Context

This updates the icons in the page and carousel menu to the new tooltips.

Note 1: Updating the tooltip for the keyboard shortcut dialog toggle is implemented in #6403.

## User-facing changes

Hover the icons in the page action menu (remove page, duplicate page, add page, undo, redo, play/stop animation) or in the carousel menu (open grid view, toggle meta-boxes) and see the difference from the old dark tooltips to the new bright tooltips.

### QA

This PR can be tested by following these steps:

1. Hover the icons in the page action menu (remove page, duplicate page, add page, undo, redo, play/stop animation) or in the carousel menu (open grid view, toggle meta-boxes)
2. Observe the new bright tooltips

### UAT

- [x] UAT should use the same steps as above.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6377

### CODE REVIEW

File: assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js [hunk 1]
```diff
@@ -27,12 +27,12 @@ import {
...
   BUTTON_TYPES,
   BUTTON_SIZES,
+  Tooltip,
 } from '../../../../design-system';
-import WithTooltip from '../../tooltip';
 
 function PageMenuButton({ children, title, shortcut, ...rest }) {
   return (
-    <WithTooltip title={title} shortcut={shortcut}>
+    <Tooltip title={title} shortcut={shortcut} hasTail>
```
Comment (miina) [inline assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: Hm, the tail of these buttons is separated with a border for some reason, whereas it doesn't happen elsewhere -- perhaps related to the placement of it and an issue with the tooltip itself?
Screenshot 2021-02-17 at 13 47 54

Comment (swissspidy) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: Probably inheriting the border from a parent? Could be solved by adding an explicit `border: none;` to `TooltipContainer`

Comment (barklund) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: I kind of hoped it was only on my machine, but I guess not.

It's not a border, it's a gap between the elements.

And I don't know why it shows here, but not in the storybook. If this offset is lowered to 7px, there's no gap. But that's a calculated value based on the height of the tail, so it doesn't really make sense:

| `top: 8px` | `top: 7px` |
|-|-|
| Screen Shot 2021-02-17 at 10 13 31 | Screen Shot 2021-02-17 at 10 13 42 |

Comment (BrittanyIRL) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: well this is odd! I will play with this a bit and find a solution. Do you want me to do this separately in a new ticket or just post up the fix here to add in this branch?

Comment (BrittanyIRL) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: fix here to merge into your branch: https://github.com/google/web-stories-wp/pull/6416

Comment (barklund) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: @BrittanyIRL, thank you so much for the quick action here! Merged!

Comment (BrittanyIRL) [reply assets/src/edit-story/components/canvas/pagemenu/pageMenuButton.js:35]: Glad I could fix it real fast! weird that it's different in the app - sorry about that!

**Review by miina — APPROVED** [review]
LGTM!

**Pull request merged**